wherehows-web/app/components/dataset-compliance-field-tag.ts

```diff
@@ -164,6 +168,37 @@ export default class DatasetComplianceFieldTag extends Component {
     return get(this, 'isIdType') && !idTypeTagHasLogicalType(get(this, 'tag'));
   });
 
+  /**
+   * Determines if the local copy of the tag has diverged from the source
+   * @type {ComputedProperty<boolean>}
+   * @memberof DatasetComplianceFieldTag
+   */
+  localTagHasDiverged = computed('sourceTag', `tag.{${changeSetReviewableAttributeTriggers}}`, function(
+    this: DatasetComplianceFieldTag
+  ): boolean {
+    const serializedSourceTagValues = Object.values(get(this, 'sourceTag'))
```

Comment:
Do we know what keys are being tested? I feel like the below code is cute for being short, but but the actual operation looping through all the keys of two objects, sorting them, converting their values to a string, and then comparing what I assume is a long string seems like overkill compared to just looping through a list of keys and comparing the values?

Reply:
~5 keys so it's definitely not a long string. I don't see this as a performance bottleneck, and trying to make this faster would be an exercise in premature optimization. I would say this lazy approach of string comparison is more declarative and reads clearer which should be the goal when performance is not an issue